Pipfile

```diff
@@ -7,9 +7,10 @@ verify_ssl = true
 
 [packages]
 ansible = "==2.8.8"
```

Comment:
Please, update to last ansible version (2.9.9?)

Reply:
2.8.12 | 2.9.10